hilti/toolchain/src/ast/operators/generic.cc

```diff
@@ -286,11 +286,13 @@ class New : public Operator {
             .result_doc = "T&",
             .ns = "generic",
             .doc = R"(
-Returns a :ref:`reference <type_reference>` to an instance of a type newly
-allocated on the heap. If ``T`` is a type, a default instance of that type will
-be allocated. If ``T`` is a constant, an instance of its type will be
-allocated and initialized with the value. Other types of expressions are not
-allowed.
+Returns a :ref:`reference <type_reference>` to an instance of a type
+newly allocated on the heap. If ``T`` is a type, a default instance of
+that type will be allocated. If the type expects any parameters, they
+must be provided through a corresponding argument tuple: ``new
+T(ARG_1,...ARG_N)``. If ``T`` is a constant, an instance of its type
```

Comment:
```suggestion
T(ARG_1, ... ,ARG_N)``. If ``T`` is a constant, an instance of its type
```

Nit: [other](https://docs.zeek.org/projects/spicy/en/latest/programming/language/types.html#enum) [places](https://docs.zeek.org/projects/spicy/en/latest/programming/language/types.html#enum) seem to use commas after the `...` too. I'd also prefer spaces around it

Reply:
Ack, consistency is good!